src/main/drivers/bus_spi.c

```diff
@@ -259,10 +261,17 @@ bool spiBusTransfer(const busDevice_t *bus, const uint8_t *txData, uint8_t *rxDa
 
 void spiSetDivisor(SPI_TypeDef *instance, uint16_t divisor)
 {
-#define BR_CLEAR_MASK 0xFFC7
+#define BR_BITS ((BIT(5) | BIT(4) | BIT(3)))
+
+    // SPI2 and SPI3 are always on APB1/AHB1 which PCLK is half that of APB2/AHB2.
```

Comment:
For all MCUs we support (and probably for entire STM32 family with two peripheral buses), only SPI2 and SPI3 run on APB1/AHB1, and everything else run on APB2/AHB2.